src/sphinx_autodoc_typehints/__init__.py

```diff
@@ -895,6 +898,9 @@ def _inject_rtype(  # noqa: PLR0913, PLR0917
 
     formatted_annotation = add_type_css_class(format_annotation(type_hints["return"], app.config))
 
+    if app.config.typehints_rtype_after_returns and r.found_return:
```

Comment:
I think general sphinx users often put the rtype after returns, so adding the ability to choose.